react-native/architecture/components/my-component/component.s.md

```diff
@@ -0,0 +1,16 @@
+# [Standard] Components folder - Component file
+
+## Standard
+
+- The file name should be `MyComponent.style.js`
+- Exports the component class by default
+
+  > **Why?:** Linked to code splitting and single responsability. Facilitates code testing.
+
+- Contains a unique `render()`
+
+  > **Why?:** Multiple renders imply multiple components
+
+- Is business logic free
```

Comment:
This is really import, but the why don't really explain it.

- When splitting container and components it is possible to test the container with unit test by passing as children a spy component that can detect props change.

Reply:
+1 pour plus d'explication.
Si je créer un composant qui à une props `subscription`, est-ce que je passe le std ?

Reply:
Need help explaining this

Reply:
- separate view logic from business logic
- make it more unit testable